client/login/wp-login/login-actions.jsx

```diff
@@ -0,0 +1,141 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import config, { isEnabled } from 'config';
+import ExternalLink from 'components/external-link';
+import Gridicon from 'gridicons';
+import { recordPageView, recordTracksEvent } from 'state/analytics/actions';
+import { resetMagicLoginRequestForm } from 'state/login/magic-login/actions';
+import { login } from 'lib/paths';
+
+export class LoginActions extends React.Component {
```

Comment:
`LoginLinks` seems like it would be more appropriate to me, as these don't have side effects (other than analytics) like the links in `TwoFactorActions` do.